ansible_base/resource_registry/resource_server.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+from typing import TypedDict
+
+import jwt
+from django.conf import settings
+
+from ansible_base.resource_registry.models import service_id
+
+
+class ResourceServerConfig(TypedDict):
+    URL: str
+    SECRET_KEY: str
+    VALIDATE_HTTPS: bool
+    JWT_ALGORITHM: str
+
+
+def get_resource_server_config() -> ResourceServerConfig:
+    defaults = {"JWT_ALGORITHM": "HS256", "VALIDATE_HTTPS": True}
+    return ResourceServerConfig(**{**defaults, **settings.RESOURCE_SERVER})
```

Comment:
Thats a lot of stars, could we just do something like `defaults.update(settings,RESOURCE_SERVER)` and pass that in as a `**` for readability?